### DISCUSSION TITLE
your 3rd party authorisation expired or unknown...

### DISCUSSION THREAD

**Discussion by manaswi** [springbok/MLM2PRO-GSPro-Connector#102] [Q&A] [answered]
...please authorize in the Rapsodo app and try again.

The connector finds the mlm, connects to it, and then gives this error message. Am i missing a step somewhere to authorize the connector in the mlm app? 

MLM2pro app running on android, connector on PC.

**Comment by springbok** [comment, marked as answer]
You need to go into the Rapsodo app and authorize 3rd party software, this needs to be done every 24 hours.

**Comment by manaswi** [comment]
OK - so this is just a quirk perhaps, but likely what is happening is that if you have not authorized either AG of E6C from the app yet (ever), then the connector fails to connect with the above message. I authorized AG this evening, and then when I tried again, the connection went through. 

How would I authorize the connector again after 24 hours since that is not in he list of 3rd party apps in the app? Perhaps it will now just go through since it has connected once.

**Comment by springbok** [comment]
You can use either one of the available apps to authorize there isn’t one specifically for the connector.
